src/common/syncjournaldb.h

```diff
@@ -282,8 +282,9 @@ class OCSYNC_EXPORT SyncJournalDb : public QObject
      */
     struct OCSYNC_EXPORT PinStateInterface
     {
+        PinStateInterface(SyncJournalDb *db);
         PinStateInterface(const PinStateInterface &) = delete;
-        PinStateInterface(PinStateInterface &&) = delete;
+        PinStateInterface(PinStateInterface &&other);
```

Comment:
why those changes ?

Reply:
Otherwise, it is impossible to use `setForPath` properly.